src/commcare_cloud/commands/ansible/service.py

```diff
@@ -58,7 +58,34 @@ def run(self, action, host_pattern=None, process_pattern=None):
         if action == 'help':
             self.print_help()
             return 0
-
+        elif action == 'logs':
+            log_locations = {
```

Comment:
rather than creating a dict here you could add this as a property on each service class